src/routes/(authenticated)/app/dashboard/+page.svelte

```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+	import { ButtonSize } from './../../../../lib/components/Base/Button/types.ts';
+	import {
+		FontWeight,
+		TextColor,
+		TextSize,
+		TextTag,
+	} from '$lib/components/Base/Text/types.ts'
+	import JoyContainer from '$lib/components/Base/Container/JoyContainer.svelte'
+	import JoyText from '$lib/components/Base/Text/JoyText.svelte'
+	import {
+		Border,
+		BorderColor,
+		BorderRounded,
+		ContainerGap,
+		ContainerPadding,
+		Justify,
+	} from '$lib/types'
+	import JoyButton from '$lib/components/Base/Button/JoyButton.svelte'
+	import { ButtonVariant } from '$lib/components/Base/Button'
+</script>
+
+<JoyContainer
+	class="w-full overflow-y-auto"
+	col={true}
+	padding={ContainerPadding.SM}
+	gap={ContainerGap.Y_LG}
+>
+	<JoyContainer class="w-full" padding={ContainerPadding.SM}>
+		<JoyContainer col={true}>
+			<JoyText tag={TextTag.PARA} size={TextSize.XS}>Thursday, August 29</JoyText>
+			<JoyText
+				tag={TextTag.H1}
+				weight={FontWeight.BOLD}
+				size={TextSize.XL_3}
+				color={TextColor.PRIMARY}>Hi John</JoyText
```

Comment:
Hello @arielrecto, can we use the `user` store from `modules/authentication` for the username. Thanks!